tla-types/src/main/scala/at/forsyte/apalache/tla/typecomp/subbuilder/LiteralAndNameBuilder.scala

```diff
@@ -23,6 +23,12 @@ trait LiteralAndNameBuilder extends UnsafeLiteralAndNameBuilder {
   /** b: Bool */
   def bool(b: Boolean): TBuilderInstruction = _bool(b).point[TBuilderInternalState]
 
+  /** root_OF_A : A */
+  def const(root: String, A: ConstT1): TBuilderInstruction = _const(root, A).point[TBuilderInternalState]
+
+  /** v : A */
```

Comment:
```suggestion
  /** v : Str */
```

Reply:
No, `v` has to have an uninterpreted type. For example `1_OF_A` (it's not `: Str`)

Reply:
ah, this was not clear to me. Shall we call the method `constParsed` or something?